### PR TITLE
Enable Gitlab's continuous integration

### PR DESCRIPTION
**Pull request by StijnVrolijk**
Enabling this by default will make us write better code, I promise!

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -48,7 +48,8 @@
         "doctrine/doctrine-fixtures-bundle": "^2.3",
         "fzaninotto/faker": "^1.5",
         "symfony/phpunit-bridge": "^3.0",
-        "phpunit/phpunit": "^5.2 || ^4.8"
+        "phpunit/phpunit": "^5.2 || ^4.8",
+        "squizlabs/php_codesniffer": "^2.6"
```
Comment (WouterSioen) [inline composer.json]: nothing in the lock file?

Comment (StijnVrolijk) [inline composer.json]: ahh didn't notice composer ran out of memory, hang on.

Comment (StijnVrolijk) [inline composer.json]: @WouterSioen added the lock file to the first commit

**Comment by WouterSioen** [comment]
:shipit:

**Comment by tijsverkoyen** [comment]
@StijnVrolijk can you fix the conflicts?

**Pull request merged**